### ISSUE TITLE
Milvus enterprise manager is not available

### ISSUE THREAD

**Issue by zxjpa** [milvus-io/milvus#4344]
条件
1.在用milvus enterprise manager连接milvus
2.使用milvus java sdk连接,并创建collection
结果
如图所示,milvus enterprise manager连接报错
![image](https://user-images.githubusercontent.com/38974941/100815641-67174000-347f-11eb-8d48-c4a2046c7ea7.png)
创建collection的参数为
```json
{
    "collectionName": "demo_file_new",
    "fieldMapList": [
      {
        "fieldName": "file_name",
        "fieldDataType": "STRING"
      },
      {
        "fieldName": "year",
        "fieldDataType": "INT8"
      }
    ],
    "vectorFieldMap": {
      "vectorFieldName": "VFM",
      "dimension": 20
    },
    "paramsInJson": {
      "segment_row_limit":  4096,
      "auto_id":  false
    }
 }
```

**Comment by shanghaikid** [comment]
@fishpenguin http 500 ~~

**Comment by sahuang** [comment]
> 创建collection的参数为
> 
> ```json
> {
>     "collectionName": "demo_file_new",
>     "fieldMapList": [
>       {
>         "fieldName": "file_name",
>         "fieldDataType": "STRING"
>       },
>       {
>         "fieldName": "year",
>         "fieldDataType": "INT8"
>       }
>     ],
>     "vectorFieldMap": {
>       "vectorFieldName": "VFM",
>       "dimension": 20
>     },
>     "paramsInJson": {
>       "segment_row_limit":  4096,
>       "auto_id":  false
>     }
>  }
> ```

Can you show the code used for creating the collection? 

String type is currently not supported in Milvus, so your collection param is invalid.

**Comment by sahuang** [comment]
Feel free to contact us if you encounter any other problems.

**Comment by zxjpa** [comment]
this is your sdk source ,please check the dataType,milvus support STRING
&nbsp;

学号：20131308059
姓名：唐瑞
QQ：2428898531
Tel：15951676228




&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Xiaohai Xu"<notifications@github.com&gt;; 
发送时间: 2020年12月2日(星期三) 下午2:33
收件人: "milvus-io/milvus"<milvus@noreply.github.com&gt;; 
抄送: "2428898531"<2428898531@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [milvus-io/milvus] Milvus enterprise manager is not available (#4344)





  
创建collection的参数为
 {     "collectionName": "demo_file_new",     "fieldMapList": [       {         "fieldName": "file_name",         "fieldDataType": "STRING"       },       {         "fieldName": "year",         "fieldDataType": "INT8"       }     ],     "vectorFieldMap": {       "vectorFieldName": "VFM",       "dimension": 20     },     "paramsInJson": {       "segment_row_limit":  4096,       "auto_id":  false     }  }
  
Can you show the code used for creating the collection?
 
String type is currently not supported in Milvus, so your collection param is invalid.

**Comment by sahuang** [comment]
SDK has string as the `DataType` but is not supported in Milvus. The next release will fix this issue and you can view the `DataType` here: https://github.com/milvus-io/milvus-sdk-java/blob/0.9.2/src/main/java/io/milvus/client/DataType.java

**Comment by zxjpa** [comment]
So if I want to use String type for save data,which DataType can I choose?





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "milvus-io/milvus"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年12月10日(星期四) 下午2:02
收件人:&nbsp;"milvus-io/milvus"<milvus@noreply.github.com&gt;;
抄送:&nbsp;"2428898531"<2428898531@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [milvus-io/milvus] Milvus enterprise manager is not available (#4344)





 
SDK has string as the DataType but is not supported in Milvus. The next release will fix this issue and you can view the DataType here: https://github.com/milvus-io/milvus-sdk-java/blob/0.9.2/src/main/java/io/milvus/client/DataType.java
